l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -97,17 +97,20 @@ def create(self, vals):
 
     @api.onchange("partner_id", "invoice_payment_term_id", "move_type")
     def _onchange_riba_partner_bank_id(self):
+        allowed_banks = (
+            self.partner_id.bank_ids or self.partner_id.commercial_partner_id.bank_ids
+        )
         if (
             not self.riba_partner_bank_id
-            or self.riba_partner_bank_id not in self.partner_id.bank_ids
+            or self.riba_partner_bank_id not in allowed_banks
         ):
             bank_ids = self.env["res.partner.bank"]
             if (
                 self.partner_id
-                and self.invoice_payment_term_id.riba
+                and self.is_riba_payment
```

Comment:
Secondo me non è nello scope del commit, o della PR in generale, ma è una modifica talmente ridotta che non bloccherei tutto per questo.